apps/styleguide/src/tables/RowEditModal.vue

```diff
@@ -131,25 +142,29 @@ export default {
       }
       this.requestMultipart(this.graphqlURL, query, variables)
         .then((data) => {
-          if (data.insert) {
-            this.success = data.insert.message;
+          console.log(JSON.stringify(data));
+          console.log(JSON.stringify(data.data.insert));
+          if (data.data.insert) {
+            console.log("insert");
+            this.success = data.data.insert.message;
+            this.$emit("update:pkey", this.value);
           }
-          if (data.update) {
-            this.success = data.update.message;
+          if (data.data.update) {
+            this.success = data.data.update.message;
           }
-          this.$emit("close");
         })
         .catch((error) => {
           if (error.status === 403) {
             this.graphqlError =
               "Schema doesn't exist or permission denied. Do you need to Sign In?";
             this.showLogin = true;
+          } else if (Array.isArray(error.errors)) {
```

Comment:
if it's an array it doesn't mean it's not empty, so assigning errors.errors[0].message could theoretically result in an error

Reply:
indeed, better safe than sorry